src/meta/Base.ts

```diff
@@ -24,7 +24,7 @@ export class Base extends Destroyable implements WidgetMetaBase {
 		return this.nodeHandler.has(key);
 	}
 
-	protected getNode(key: string | number): Element | undefined {
+	public getNode(key: string | number): Element | undefined {
```

Comment:
Why did you need to make this `public`?

Reply:
The `WebAnimation` meta stubs it in unit tests, and the updated version of sinon complained if it wasn't `public`

Reply:
Ah that's a little annoying :(

Reply:
yeah :/

Reply:
I think we need to raise an issue to address the `WebAnimation` tests rather than doing this here. Should probably have bumped `sinon` in it's own PR.